crates/fluvio-spu/src/services/public/offset_request.rs

```diff
@@ -55,7 +55,7 @@ pub async fn handle_offset_request(
                     match fetch_consumer_offset(&ctx, topic, *partition, consumer_id).await {
                         Ok(Some(consumer_offset)) => {
                             debug!(consumer_id, consumer_offset, "consumer offset");
-                            partition_response.start_offset = consumer_offset + 1;
+                            partition_response.consumer_offset = Some(consumer_offset + 1);
                         }
```

Comment:
should have integration test

Reply:
I added some tests at `crates/fluvio-test/src/tests/consumer_offsets/` that should cover it

Reply:
Or do you mean another kind of integration test?

Reply:
That's E2E tests, good job on there.

The integration test should go here: https://github.com/infinyon/fluvio/tree/master/crates/fluvio-spu/src/services/public/tests.

Reply:
I added integration test